### DISCUSSION TITLE
Allow old "mirror" functionality again

### DISCUSSION THREAD

**Discussion by Qix-** [community/community#5855] [Other Feature Feedback, Questions, & Ideas]
Hey there,

A while back Github had the ability to import a mirror of an external Git repository, and to have them synced up automatically (a pull every once in a while). This functionality doesn't seem to exist anymore, with the "import repository" only allowing a one-shot import as I understand it.

Would Github consider re-enabling the ability to add repository mirroring?

Thanks!

**Comment by ancarda** [comment]
You can still setup new mirrors, however you have to do this via a support ticket. There isn't a UI for creating or managing mirrors.

A UI for this would be great, thank you!
